templates/config.j2

```diff
@@ -58,15 +58,11 @@ providers:
 {% endif %}
     defaultRule: {{ traefik_docker_default_rule }}
 
-  ansible.builtin.file:
```

Comment:
file would be right here, just ansible.builtin. got to be removed :)

Reply:
Gotcha :sweat_smile: